assets/variables.scss

```diff
@@ -3,9 +3,17 @@
 // The variables you want to modify
 // $font-size-root: 20px;
 
+@import '~vuetify/src/styles/styles.sass';
+
 // Globals
 $body-font-family: 'Noto Sans',
 'Roboto',
 sans-serif !default;
 $font-size-root: 14px;
 $input-font-size: 14px;
+
+$material-dark: map-merge($material-dark, ('navigation-drawer': #1a2932,
```

Comment:
What's the difference from the way I did here? https://github.com/jellyfin/jellyfin-vue/pull/377/files#diff-227a90a8ae75d004c6de80d5393f195c4213eb42f7740de16aaec04e72cbc745R10

Also is it possible to add the background color here too, so the dark mode sync PR works? :)

Reply:
I'm not sure what the difference is, honestly :smile: It just seemed more proper to merge the new values with the old ones.

Reply:
Fair enough :)